py_neuromodulation/nm_stream.py

```diff
@@ -134,23 +140,38 @@ def _run(
                 data_batch.astype(np.float64)
             )
             if is_stream_lsl:
-                feature_dict["time"] = time_[-1]
                 if self.verbose:
                     if last_time is not None:
                         logger.debug("%.3f seconds of new data processed", time_[-1] - last_time)
```

Comment:
I don't think DEBUG level messages should depend on verbose, since DEBUG usually is disabled by default unless you set the level of logging  to DEBUG explicitely

Reply:
Good catch